Release_Guideline.md

```diff
@@ -0,0 +1,27 @@
+This document captures guidelines to follow when considering a new release of the BIDS specification.
+
+BIDS has generally followed a semantic versioning (semver) approach.
+The canonical semver text can be found at [https://semver.org/](https://semver.org/), and an adaptation to documents can be found at [https://semverdoc.org/](https://semver.org/).
+A specification falls somewhere between these two, so we should consider carefully the rules.
+
+In both cases, edition numbers are specified as MAJOR.MINOR.PATCH, and the difference is in the rules for incrementing each edition number.
+
+In BIDS, we have considered a major release version (the next being 2.0.0) to indicate backwards incompatible changes, and minor and patch releases must be backwards compatible.
+
+Once a decision for a release has been established, the rules of [decision-making](DECISION-MAKING) govern the mechanism of doing the release (i.e., waiting 5 business days to merge).
+
+**When it is time for a minor (1.X) release?**
+
+This release should be reserved for when a modality (BEP) has been merged into the specification.
+Special cases can override this and it is up to the BIDS Maintainers Group to decide this given approval by the BIDS Steering Group.
```

Comment:
Given that we're still awaiting approval on the governance PR, maybe we should just be vague here and say that something like :

```suggestion
Special cases can override this at the discretion of BIDS Maintainers.
```

Unless you want to wait and merge this after governance !

Reply:
I think this should be merged before governance and then revised and iterated upon :-)

+1 for your suggestion to remain a bit vague

Reply:
Sorry, was going through top-to-bottom, so I already suggested something that's a bit more prescriptive than restrictive, which I think fits fairly well with what you're getting at, @emdupre, but needs a slightly different follow-up.

```suggestion
The BIDS Maintainers have discretion to identify other cases justifying a minor release.
```

Reply:
+1 merging this before the governance. The BIDS Maintainers can be changed to BIDS Maintainers Group at that time. What further follow up did you have in mind? It flows nicely from the minor to patch.